pkg/resources/nifi/pod.go

```diff
@@ -141,8 +141,11 @@ func (r *Reconciler) pod(node v1.Node, nodeConfig *v1.NodeConfig, pvcs []corev1.
 				RunAsUser:    nodeConfig.GetRunAsUser(),
 				RunAsNonRoot: func(b bool) *bool { return &b }(true),
 				FSGroup:      nodeConfig.GetFSGroup(),
+				SeccompProfile: &corev1.SeccompProfile{
+					Type: corev1.SeccompProfileTypeRuntimeDefault,
```

Comment:
This should probably be part of the `NodeConfig` as well, like the above settings. Defaulting to the runtime default.